### DISCUSSION TITLE
copy all output to my Soundcard to bluealsa too

### DISCUSSION THREAD

**Discussion by novazur972** [Arkq/bluez-alsa#634] [General]
Hi,

I've already spent a lot of time looking for how to do it, but I can't figure it out on my own. I really need your skills.
I use the alsaequal and dmix plugin for output to my sound card with the configuration below:

```
defaults.pcm.card 0 # or better "PCH"
defaults.pcm.device 0
ctl.equal {
         type equal;
}
pcm.plugequal {
         type equal;
         slave.pcm "plug:dmix";
}
pcm.asymed {
         asymmetric type
         playback.pcm "plugequal"
         capture.pcm "hw:3.0"
}
pcm.!default { type plug slave.pcm "asymed" }

```
I use bluealsa to get the sound output (with or without equal) to my bluetooth headset, and it works fine with the following setup:

```
ctl.equal {
type equal;
}
pcm.plugequal {
type equal;
slave.pcm bt;
}
pcm.equal{
plug type;
slave.pcm plugequal;
}

pcm.bt {
         plug-type
         slave.pcm bluealsa
}
ctl.bt {
         blue alsa type
}

pcm.asymed {
         asymmetric type
         playback.pcm "plugequal"
         capture.pcm "hw:3.0"
}

pcm.!default { type plug slave.pcm "asymed" }
```
But what I would have liked to do is that all the sound goes out on my sound card AND on the bluetooth headset if it is connected, so a copy to the bluetooth.
I tried to follow many documentations but without success.
I have the feeling that it is necessary to use multi, but will not I lose the use of the dmix?
Your help would be invaluable, especially with a specific configuration example that does what I need.
(If there are superfluous elements in my configurations, feel free to delete them)
Thanks in advance
(google translate)

**Comment by borine** [comment]
I think that what you are trying to achieve is not possible with ALSA plugins. As noted in the [wiki page](https://github.com/arkq/bluez-alsa/wiki/Using-BlueALSA-with-other-ALSA-plugins), the BlueALSA PCM plugin can be opened by only one application at a time, so if you place a `multi` plugin in front of it then that `multi` can also be opened by only one application at a time.

**Reply by novazur972** [reply]
Ok, I understand. Thank you.
Do you know if pulseaudio is the solution ?
I don't know anything about it.
I have always postponed the moment when I should go there. I hate overlays. Pulseaudio is an ALSA overlay, isn't it?
But if it's the solution to having a more agile system than it is with ALSA alone, maybe I'll have to switch to pulseaudio.
